behat.yml.dist

```
@@ -56,6 +57,32 @@ default:
                                 unexpectedAlertBehaviour: accept
                                 goog:chromeOptions:
                                     w3c: false # https://github.com/Sylius/Sylius/issues/10561
+                panther:
+                    panther:
+                        options:
+                            browser: 'chrome'
+                            webServerDir: '%paths.base%/public'
+                            external_base_uri: https://127.0.0.1:8080/'
+                        kernel_options:
+                            APP_ENV: test_cached
+                            APP_DEBUG: false
+                        manager_options:
+                            connection_timeout_in_ms: 5000
+                            request_timeout_in_ms: 120000
+                            capabilities:
+                                browserName: chrome
+                                browser: chrome
+                                extra_capabilities:
+                                    chromeOptions:
+                                        args:
+                                            - "--window-size=2880,1800"
+                                            - "--headless"
+                                            - "start-fullscreen"
+                                            - "start-maximized"
+                                            - "no-sandbox"
+                                acceptSslCerts: true
+                                acceptInsecureCerts: true
+                                unexpectedAlertBehaviour: accept
```

After many attempts, xdebug step by step and a lot of articles reading, this config can be reduced to :

```suggestion
                panther:
                    panther:
                        manager_options:
                            connection_timeout_in_ms: 5000
                            request_timeout_in_ms: 120000
                            capabilities:
                                acceptSslCerts: true
                                acceptInsecureCerts: true
                                unexpectedAlertBehaviour: accept
```

Most of the configurations were default one, the rest can only be added through env vars like this :

```env
PANTHER_CHROME_ARGUMENTS=--window-size=2880,1800 --start-fullscreen --start-maximized --no-sandbox
```
This can also be use to see on your own machine what Chrome is displaying :

```env
PANTHER_NO_HEADLESS=true
```